runtime/darwinia-parachain/src/pallets/remote_governance.rs

```diff
@@ -0,0 +1,46 @@
+// --- paritytech ---
```

Comment:
@jiguantong @HackFisher Please help check the config for this pallet.

Reply:
looks ok